travis_configure.sh

```diff
@@ -104,7 +131,14 @@ run_tests()
             print_error_msg "Invalid build configuration"
             return 1
         fi
-
+    elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        autoreconf -fi
+        ## Pass the root of the openssl installation directory
```

Comment:
Here also a double `##` is used.
I don't mind using single `#` or double `##`, it's just not consistent with the other parts I see here in this file.